package.json

```diff
@@ -39,7 +39,8 @@
     "react-spring": "npm:@react-spring/web@9.0.0-rc.3",
     "react-use": "^15.3.4",
     "react-use-measure": "^2.0.2",
-    "turndown": "^6.0.0"
+    "turndown": "^6.0.0",
+    "use-media": "^1.4.0"
```

Comment:
[nit] It looks like we have `react-use` installed, which also comes with its own [`useMedia`](https://github.com/streamich/react-use/blob/master/docs/useMedia.md) hook. If you want to save on an extra dependency, you could use the `react-use` version.

Reply:
That sounds good to me! I'll make that change